plugin/oauth2/install.php

```diff
@@ -0,0 +1,8 @@
+<?php
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [End of line character is invalid; expected "'\n'" but found "\n"](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5679956)